lib/shared.bash

```diff
@@ -7,6 +7,12 @@ function strip_quotes() {
 function get_secret_value() {
   local secretId="$1"
   local allowBinary="${2-}"
+  local region="${3-}"
```

Comment:
Should these have `:`'s? I.e. `region=${3:-}`?

Reply:
They can, the difference is null vs empty handling. I'll update 😁